src/raven.js

```diff
@@ -759,7 +760,14 @@ function makeRequest(data) {
     var img = newImage(),
         src = globalServer + authQueryString + '&sentry_data=' + encodeURIComponent(JSON.stringify(data));
 
-    img.crossOrigin = 'anonymous';
+    if ('crossOrigin' in globalOptions) {
```

Comment:
>  I added this check because I saw that in the testing, there's assumptions that you can override the globalOptions object completely (there were direct assignments to globalOptions).

Ahh, this is _only_ being done in tests. `globalOptions` doesn't exist in a production build in the public scope, so there's no way to clobber it.